usecase/pet.go

```diff
@@ -21,17 +21,28 @@ func (c *PetUseCase) FindById(userID uniqueEntity.ID) (*entity.Pet, error) {
 	return nil, nil
 }
 
-func (c *PetUseCase) Update(petID string, userID string, petToUpdate *entity.Pet) (err error) {
+func (petUseCase *PetUseCase) Update(petID string, userID string, petToUpdate *entity.Pet) (err error) {
 
 	updateValues := map[string]interface{}{}
 
-	if c.isValidPetSize(petToUpdate) {
-		updateValues["size"] = &petToUpdate.Size
+	if petUseCase.isValidPetSize(petToUpdate) {
+		updateValues["size"] = petToUpdate.Size
 	} else {
 		return errors.New("the animal size is invalid")
 	}
 
-	err = c.repo.Update(petID, userID, updateValues)
+	if petUseCase.isValidWeight(petToUpdate) {
+		updateValues["weight"] = petToUpdate.Weight
+		updateValues["weight_measure"] = petToUpdate.WeightMeasure
```

Comment:
A chave do map precisa ser do mesmo nome da coluna no banco. 
```suggestion
		updateValues["weightMeasure"] = petToUpdate.WeightMeasure
```